assets/containerisableComponents/contributionSelection/contributionSelection.jsx

```diff
@@ -49,8 +50,14 @@ function ContributionSelection(props: PropTypes) {
 
   const modifierClass = getContributionTypeClassName(props.contributionType);
 
+  let oneOffModifierClass = '';
+  if (props.oneOffOnlyVariant && props.oneOffOnlyVariant === 'oneOffOnly') {
```

Comment:
I think this can just be `props.oneOffOnlyVariant === 'oneOffOnly'`, you shouldn't need to check for the existence of the string since the type only allows one of three strings to be passed here.